sw/device/tests/sim_dv/pwrmgr_deep_sleep_all_wake_ups.c

```diff
@@ -78,14 +78,20 @@ typedef struct test_wakeup_sources {
 } test_wakeup_sources_t;
 
 enum {
-  // test wake up input 0 : sysrst_ctrl
+  // Test wake up input 0 : sysrst_ctrl
   kTestSysrstCtrlWakeupSource = 0,
 
-  // test wake up input 1 : adc_ctrl
+  // Ttest wake up input 1 : adc_ctrl
```

Comment:
```suggestion
// Test wake up input 1 : `adc_ctrl`.
```

Reply:
@engdoreis  My apology, this file is updated in separate PR(https://github.com/lowRISC/opentitan/pull/12937)
all these comments are addressed. Didn't mean to show this PR.